refactor: modularizado o código

EleyOliveira opened the pull request:
separado as rotas e controllers

EleyOliveira committed:
refactor: modularizado o código

separado as rotas e controllers

eleyoliveira2 commented on main.go:
O HandleRequest precisa ser chamado antes de inicializar o servidor, caso contrário aparecerá o erro 404.

EleyOliveira replied:
resolvido

eleyoliveira2 reviewed (changes requested):
Alterar o arquivo main.

EleyOliveira committed:
fix(main): alterado o arquivo main

resolvido o erro 404 alterando a chamada das rotas antes de inicializar o servidor

eleyoliveira2 reviewed (approved):
Certo
